### PR TITLE
Better setup instructions

### PR DESCRIPTION
**Pull request by audgster**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,159 @@
...
+Node comes with npm, which is a package manager. Globally install the react-native cli by running
+```
+npm install -g react-native-cli
+```
+
+Note, this project requires npm to be version 6.4.1 or greater.
+
+#### 3 - Install Android Studio (for android emulation)
+
+Follow the guide [here](https://developer.android.com/studio/install) to install android studio on your mac. You will also need to [create an emulator](https://www.embarcadero.com/starthere/xe5/mobdevsetup/android/en/creating_an_android_emulator.html)
```
Comment (renana) [inline README.md]: your Windows.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,159 @@
-## Fearlessly Girl Mobile App
+# Fearlessly Girl Mobile App
 
-This project contains the Fearlessly Girl Mobile App.
-
-Below you'll find information about installing and running this in your local environment. You will also see how to contribute to the project and the practices and patterns that are in use with examples of how to implement some interactions.
+FearlesslyGirl is an organization dedicated to making the world a kinder place by creating healthy discourse between highschool aged girls.
```
Comment (renana) [inline README.md]: Can we add here a link to FearlesslyGirl website?

File: README.md (hunk 1 above)
Comment (audgster) [reply README.md]: ooops thanks!

File: README.md (hunk 2 above)
Comment (audgster) [reply README.md]: yes! thanks!

**Pull request merged**